modules/cnf-fast-event-notifications-api-refererence.adoc

```diff
@@ -289,3 +292,114 @@ $ oc logs -f linuxptp-daemon-cvgr6 -n openshift-ptp -c cloud-event-proxy
    }
 }
 ----
+
+== /api/cloudnotifications/v1/cluster/node/<node_name>/<event_type>/CurrentState
+
+[discrete]
+=== HTTP method
+
+`GET api/cloudNotifications/v1/cluster/node/<node_name>/sync/ptp-status/lock-state/CurrentState`
+
+`GET api/cloudNotifications/v1/cluster/node/<node_name>/sync/sync-status/os-clock-sync-state/CurrentState`
+
+`GET api/cloudNotifications/v1/cluster/node/<node_name>/sync/ptp-status/ptp-clock-class-change/CurrentState`
+
```

Comment:
@aneeshkp - I added the Get request for each supported event here. Hope that this is clear to the user?

Reply:
It says `<resource_address>` in the section heading now. But in the GET request examples, I have expanded the route to show the format `cluster/node/<node_name>`. Let me know if I should change this to resource address too

Reply:
looks good